### DISCUSSION TITLE
讨论时的常见疑惑

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#775] [公告]
### Issue 会在多长时间内处理？
新 Issue 会在 **两天内** 得到初步审核，并被打上其他标签。
如果被标记为 `处理中` ，那么它很可能会在下一次 PCL 更新前被处理。PCL 每次更新间隔 **2 周至 1 个月** ，我会在每次更新前集中处理 `处理中` 的 Issue。
所以，你可能会看到屯着一大堆 `处理中` 的 Issue，直到下次更新时才被一次性批量处理……这是正常的。

----
### Collaborator 是 PCL 官方吗？
不是。Collaborator 是 **无偿维护社区的志愿者**。他们可能来自于各行各业，可能是职业的工程师，也可能只是抽出课余时间来帮帮忙的普通的学生。所以，Collaborator 可能无法及时处理 Issue，但他们本就没有处理 Issue 的 **责任**，本来就只是来顺手帮帮忙的！
他们拥有社区相关的权限，但并不属于 PCL 的开发组、不参与 PCL 的开发，也不代表 PCL 官方的意见。
如果你认为某个 Collaborator 的处置存在问题，可以 @LTCatt 让我来看看。
你也可以通过积极协助处理 Issue 和 Discussion 来成为 Collaborator，详见 #477 的介绍！

----
### 为什么我的改进建议会被拒绝？它真的能改进 PCL 啊！
PCL 的开发仅由我一人业余完成，这代表我能花在 PCL 上的时间是 **非常有限** 的，我需要把有限的开发时间花在改进更大、收益更高的地方。所以，当有许多改进建议摆在我面前时，我只能无情地选择 **能用上它的人最多的、制作耗时最少的、对体验优化最大的** 功能。
是的，我知道很多建议真的能改进 PCL，但在时间压力下，它的制作优先度会排得很低——以至于几年都轮不到它。因此，优先度太低的建议就会直接被拒绝。因为即使不拒绝，你也不太可能能在 MC 凉掉之前看到它实装……
这不代表我认为它没用或者不算改进，而是我需要把时间花在更有价值的功能上。
抱歉啦。

----
### 扩展阅读
[常见&难检反馈及问题列表](https://github.com/Hex-Dragon/PCL2/discussions/1930)

**Comment by ABCcomeon** [comment]
能问一下联机什么时候完成吗

**Reply by LTCatt** [reply]
没有这个预计，不要瞎说，求求了

**Comment by so1ve** [comment]
增加提出issue或discussion的规范？比如BUG问题需要提供日志以及崩溃截图等

**Reply by so1ve** [reply]
还有FAQ？

**Reply by WForst-Breeze** [reply]
现在的 Issue Template 应该就属于是规范了，里面的相关事项写的都很清楚。我个人认为，不论如何总是避免不了会有朋友未能上传有效信息或者不认真看检查项的，所以我认为可能即使出了一篇规范，实际影响也不会很大qwq

**Reply by so1ve** [reply]
合理（）不管在什么仓库下面都会有人将issue模板当成空气

**Comment by Maimengtat** [comment]
1.19.2老是报错，修一下

**Reply by WForst-Breeze** [reply]
请提交 Issue 或 Discussion 并提供相关信息以使我们确定问题存在

**Comment by 149441** [comment]
[Uploading 错误报告-2023-04-05_15.58.42.zip…]()
这个咋办

**Reply by so1ve** [reply]
一，你文件没传上来
二，有问题请发一个新的discussion，确认是bug后再发issue

**Reply by WForst-Breeze** [reply]
请在[此处](https://github.com/Hex-Dragon/PCL2/discussions/categories/%E6%8F%90%E9%97%AE)提问

**Comment by bangbadan** [comment]


**Reply by iPodtouch6557** [reply]
这里不是你提优化的地，请提交issue

**Reply by bangbadan** [reply]
嗯，确实是没有找对地方，很抱歉

**Comment by iPodtouch6557** [comment]
我认为……可以列一下常常重复的问题……？
edit：已列，请见 #1930

**Comment by genyigeyuehui** [comment]


**Reply by iPodtouch6557** [reply]
~这问题似曾相识啊~无论如何请你去交issue，这个是指引你如何交issue，不是反馈问题的，我真的要……（）

**Reply by WForst-Breeze** [reply]
正常情况，Bugjump 的 Bug，大部分买一赠一的帐号都有这个问题，详见 #1299。非 PCL2 问题
